cmd/queue/main.go

```diff
@@ -307,8 +329,43 @@ func main() {
 		logger.Fatalw("Failed to parse localhost URL", zap.Error(err))
 	}
 
+	queueProxyL3 := fmt.Sprintf("%s:%d", env.ServingPod, networking.ServiceHTTPPort)
+	zipkinEndpoint, err := zipkin.NewEndpoint(env.ServingPod, queueProxyL3)
+
+	if err != nil {
+		logger.Fatalw("Unable to create tracing endpoint", zap.Error(err))
+		return
+	}
+
+	oct := tracing.NewOpenCensusTracer(
+		tracing.WithZipkinExporter(tracing.CreateZipkinReporter, zipkinEndpoint),
+	)
+
+	enableVal, err := strconv.ParseBool(env.TracingConfigEnable)
+	if err != nil {
+		logger.Infow("Error parsing tracing enable var", zap.Error(err))
+	}
+	debugVal, err := strconv.ParseBool(env.TracingConfigDebug)
+	if err != nil {
+		logger.Infow("Error parsing tracing debug var", zap.Error(err))
+	}
+	sampleVal, err := strconv.ParseFloat(env.TracingConfigSampleRate, 64)
```

Comment:
As commented by @vagababov above, please push all of the typing into the config struct above, the library will do the parsing/erroring for you.